src/main/resources/org/jenkinsci/plugins/oic/OicServerManualConfiguration/config.jelly

```diff
@@ -0,0 +1,36 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+
+    <f:entry title="${%TokenServerUrl}" field="tokenServerUrl">
+        <f:textbox />
+    </f:entry>
+
+    <f:entry title="${%TokenAuthenticationMethod}" field="tokenAuthMethod">
+        <f:radioBlock title="${%Basic}" name="tokenAuthMethod"
+            checked="${instance.tokenAuthMethod == 'client_secret_basic'}" value="client_secret_basic" inline="true" help="${null}"/>
+        <f:radioBlock title="${%Post}" name="tokenAuthMethod"
+            checked="${instance.tokenAuthMethod == null || instance.tokenAuthMethod == 'client_secret_post'}" value="client_secret_post" inline="true" help="${null}"/>
```

Comment:
set the default here for new installs with the `== null` check (slightly fewer clicks for the defaults)